script/mafia

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -eu
 
+MAFIA_HOME=${MAFIA_HOME:-$HOME/.mafia}
+
```

Comment:
`:` Is a shell builtin (`man builtins`) that is a command that always returns true, it is effectively a symbolic version of `true`, but by convention in most shell (well BSD world anyway) to read as "evaluate all of my arguments, ignore them and do nothing", the "evaluate all of my arguments" actually being just normal shell expansion/evaluation nothing really specific to `:`. After that it is just normal shell expansion, `${_:=_}` as in assign if not set, rather than `${_:-_}` as in return if not set. But you need the `:` otherwise the shell would run the output of `${}` as a command and you just want to ignore it.